Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -507,6 +507,18 @@ public static StatusScreen getStatus(Nation nation, CommandSender sender) {
 						.append(Component.newline())
 						.append(translator.component("status_hover_click_for_more"))),
 				ClickEvent.runCommand("/towny:nation enemylist " + nation.getName()));
+		
+		// Sanctioned Towns [3]: Prague, Berlin, Vienna
+		List<String> sanctionedTowns = getFormattedNames(nation.getSanctionedTowns());
+		if (sanctionedTowns.size() > 10)
+			shortenOverLengthList(sanctionedTowns, 11, translator);
+		
+		if (sanctionedTowns.size() > 0)
+			screen.addComponentOf("sanctionedtowns", colourHoverKey(translator.of("status_nation_sanctioned_towns")),
+				HoverEvent.showText(TownyComponents.miniMessage(getFormattedStrings(translator.of("status_nation_sanctioned_towns"), sanctionedTowns, nation.getSanctionedTowns().size()))
+					.append(Component.newline())
+					.append(translator.component("status_hover_click_for_more"))),
+				ClickEvent.runCommand("/towny:nation sanctiontown list"));

```

Comment:
This should also take in the nation name so you're not only seeing your own nation's sanctioned towns.

Reply:
I'll fix it when I wake up